model/logs/mobile-crashes.yaml

```diff
@@ -0,0 +1,42 @@
+groups:
+  - id: mobile-crash-event
+    type: event
+    brief: >
+      This document defines attributes for crashes represented using Log Record Events.
+      `event.domain` is 'mobile' and `event.name` is 'crash
+    attributes:
+      - id: platform
+        requirement_level: "required"
+        brief: >
+            This attribute represents the state the application has transitioned into at the occurrence of the event.
+        type:
+          allow_custom_values: false
+          members:
+            - id: android
+              value: 'android'
+              brief: 'Value if crash happened on android device'
+            - id: ios
+              value: 'ios'
+              brief: 'Value if crash happened on ios device'
+      - id: isAppInForeground
+        requirement_level: "required"
+        brief: >
+          This attribute indicates whether the app was in the foreground or not during the crash.
+        type:
+          allow_custom_values: false
+          members:
+            - id: true
+              value: 'true'
+              brief: 'Value if crash happened in foreground'
+            - id: false
+              value: 'false'
+              brief: 'Value if crash did not happen in foreground'
+      - id: data
+        requirement_level: "required"
+        type: attribute_group
+        brief: >
+          'event.data' in the case of mobile crashes will be an attribute that contains nested attributes.
+          Android crashes will contain 'stacktrace', 'thread', and 'throwableMessage'
+          On the iOS side, it will contain 'crashReport' containing the contents from PLCrashReporter
```

Comment:
Why the content of PLCrashReporter and not an Apple style crash file? which can be generated not only from PLCrashreporter but from others like KSCrash. Apart from supporting other crash handlers it would also support existing tools for symbolication

Reply:
We do actually use PLCrashReporter with the Apple symbolication style.

Reply:
May be the description can be made generic to incorporate all. Something like - "On iOS, it will contain 'crashReport' (obtained from iOS crash reporters like PLCrashReporter, KSCrash etc.)"